### DISCUSSION TITLE
New installation on Raspberry Pi Zero W failing

### DISCUSSION THREAD

**Discussion by g4n-g4n** [mKeRix/room-assistant#1203] [Q&A]
Hi guys,

Have a few problems in installing room-assistant into Raspberry Pi Zero W's recently. I've installed 4-5 in the past a few months ago, but had to reinstall them and I'm usually getting stuck at the math line. Installing it in verbose states that all the dependencies  (as in a lot of lines) fail.

Anyone have updated steps other than the ones on the room-assistant.io site?

Thanks.

**Comment by Duke-Box** [comment]
Hi,

I also had this issue when installing on a Pi zero W.
It appeared to stop at the mathjs line.
I had several attempts and also used the verbose command line.
If you are patient, you can see it fallback to build. And then if you are really patient you can see it rebuilding the component and then the installation succeeds.

On my pi zero, this took 47 mins from start of the RA installation to getting the success banners.
Hope this helps
